### DISCUSSION TITLE
new release

### DISCUSSION THREAD

**Discussion by EfremBraun** [pyqtgraph/pyqtgraph#2171] [General]
I'm wondering if a new release is planned for anytime soon. I really like #2011 and I'm hoping to be able to use a version of pyqtgraph that has it in the `requirements.txt` file for my code.

**Comment by j9ac9k** [comment]
Hi @EfremBraun 

I had been hoping to do a new release (0.12.4) early December, but with having visitors for the holidays, and now post-holiday period I have had some personal events consume my time that is normally allocated towards my open source work (this can probably be observed by my github activity chart on my profile page).  I suspect some of the other maintainers are in a similar situation.

I can say, I was hoping to merge #1359 before doing the next release; a lot of work has gone into that PR over the course of the last year+ and it's a highly requested feature. 

If you have the time/capability/motivation to take a look at that PR, there are a couple of outstanding issues in the first post.  If you could chime in/make suggestions there I think that would go a long ways.

Sorry I don't have a better answer!

**Reply by EfremBraun** [reply]
Gotcha. Actually, I was one of the people who requested multi axis plots! Happy to see it's being implemented!

**Reply by j9ac9k** [reply]
I think developing multi-axis plots is one of those rights of passage for developers using pyqtgraph.  One of my first projects involving pyqtgraph involved multi-axis plots 😆

**Comment by temeddix** [comment]
0.12.4 has a ton of new features and bug fixes. It's an awesome work. I've been waiting it for it for more than four months and hope it would roll out soon.

**Reply by EfremBraun** [reply]
See https://github.com/pyqtgraph/pyqtgraph/pull/1359#issuecomment-1048063574.

**Reply by j9ac9k** [reply]
I've been a little overwhelmed in my non-OSS life and have been taking an extended break from my pyqtgraph work.

Anyway, the release is close https://github.com/pyqtgraph/pyqtgraph/pull/2209

**Reply by j9ac9k** [reply]
And we've released it

**Reply by temeddix** [reply]
Thank you for the notification!
